test/evergreen.yml

```diff
@@ -1990,6 +1990,48 @@ tasks:
           # No of processes to run in the background
           no_of_procs: 10
 
+  # The task name is ppc-zseries because this task will be used in both buildVariants
+  - name: format-stress-ppc-zseries-test
+    # Set 2.5 hours timeout (60 * 60 * 2.5)
+    exec_timeout_secs: 9000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger with builtins"
+      - func: "format test script"
+        vars:
+          format_test_setting: ulimit -c unlimited
```

Comment:
Retain the comment explaining the purpose of this setting would be useful.

Reply:
Fixed